libs/openFrameworks/3d/ofEasyCam.cpp

```diff
@@ -14,9 +14,15 @@ ofEasyCam::ofEasyCam(){
 	lastTap	= 0;
 	lastDistance = 0;
 	drag = 0.9f;
-	sensitivityRot = 1.0f;//when 1 moving the mouse from one side to the other of the arcball (min(viewport.width, viewport.height)) will rotate 180degrees. when .5, 90 degrees.
-	sensitivityXY = .5f;
-	sensitivityZ= .7f;
+ 
+	sensitivityRotX = 1.0f;
+    sensitivityRotY = 1.0f;
+	sensitivityRotZ = 1.0f;
+	
+	// these values now default to 1.0, but are scaled by previous defaults in updateMouse() for consistency
```

Comment:
Can probably remove this comment if you've documented the behavior in the PR and in the header.  Since it references (now) old code, it isn't particularly helpful.